src/lib/createWorkerBlobUrl.ts

```diff
@@ -17,10 +18,15 @@ import remoteDepsParser from './remoteDepsParser'
  * .catch(postMessage(['ERROR', error])"
  */
 const createWorkerBlobUrl = (
-  fn: Function, deps: string[], transferable: TRANSFERABLE_TYPE,
+  fn: Function,
+  deps: string[],
+  transferable: TRANSFERABLE_TYPE,
+  localDeps: () => unknown[],
 ) => {
+  const [context] = isoworker.createContext(localDeps)
```

Comment:
`createContext` returns a few other things, including some transferables that should be sent over. You should assign the received values of the transeferables to the global environment.